getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1591,6 +1591,24 @@ You can also get the signal's argument once it's emitted by an object:
     # Wait for when any node is added to the scene tree.
     var node = yield(get_tree(), "node_added")
 
+If there is more than one argument, ``yield`` returns an array containing
+the arguments::
```

Comment:
```
If there is more than one argument, ``yield`` returns an array containing
the arguments::
```

Might be:

```
If there is more than one argument, ``yield`` returns an array containing
the arguments.

::
```

Reply:
That's equivalent, the `::` at the end of a sentence will become `:` + a codeblock.
https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/basics.html#literal-blocks

I think we tend to use both forms in the docs currently, we haven't really defined that one should be used over the other (yet).

Reply:
Ah well not strictly equivalent as your suggestion ends with a dot, not colon. I don't have a strong opinion there.

Reply:
It's mixed in the docs. I personally think that using the colon is better when you refer directly to the code (what I do in that passage). Otherwise the dot should be used, but that's just personal preference.